vector/src/main/java/im/vector/app/features/home/room/list/RoomCategoryItem.kt

```diff
@@ -46,14 +46,18 @@ abstract class RoomCategoryItem : VectorEpoxyModel<RoomCategoryItem.Holder>() {
             DrawableCompat.setTint(it, tintColor)
         }
         holder.unreadCounterBadgeView.render(UnreadCounterBadgeView.State(unreadNotificationCount, showHighlighted))
-        holder.titleView.setCompoundDrawablesWithIntrinsicBounds(null, null, expandedArrowDrawable, null)
         holder.titleView.text = title
+        with(holder.counterView) {
+            text = ""
```

Comment:
does this reset the counter amount?

Reply:
Here it does not display the counter at all. For what I understood of the code, this item may be used in a screen to add a room but I didn't succeed to find where in the app. I didn't reach creation of this item in debug mode. So, I just supposed we don't need a counter in this use case. Do you know how to open the screen which uses this item ?

Reply:
from what I can tell `RoomCategoryItem/RoomCategoryItem_` is only used by the `AddRoomListController` which is used by the `SapceAddRoomFragment`

which I believe is this screen...

![Screenshot_20220210_102639](https://user-images.githubusercontent.com/1848238/153387981-67838c1c-be28-4faf-8933-474d8dff51f4.png)

to get here I did, `menu drawer` -> `add space` -> `private, just me`, this screen should also be accessible by editing an existing space to add/remove rooms

Reply:
if we don't want to have a counter here, it could be nice to extract a constant for the empty copy with a name like 

```kotlin
const val NO_COUNTER = ""
```

Reply:
I will try to add the counter in this screen as well to have consistent UI.